src/main/java/javaClasses/homework_4/stepan_alvajyan/Developer.java

```diff
@@ -0,0 +1,18 @@
+package javaClasses.homework_4.stepan_alvajyan;
+
+public class Developer extends Staff {
+    String techStack;
```

Comment:
In classes we always want to have private fields, to protect our  object from external changes. It should be changed/accessed only via setters and getters. Please change access modifier to private, in other classes too.